lib/ansible/plugins/action/reboot.py

```diff
@@ -42,18 +42,28 @@ class ActionModule(ActionBase):
 
     DEPRECATED_ARGS = {}
 
+    BOOT_TIME_COMMANDS = {
+        'linux': DEFAULT_BOOT_TIME_COMMAND,
```

Comment:
I think we can omit all the entries that reference `DEFAULT_BOOT_TIME_COMMAND` just point to that in the `get()` calls.